README.md

```diff
@@ -6,7 +6,7 @@
     <br>
 </p>
 
-This library provides [RBAC] (Role-Based Access Control) library.
```

Comment:
@samdark should we rename this to "package", which is more convinient, or leave "library" according to [docs](https://github.com/yiisoft/docs/blob/master/000-packages.md)?